heron/statemgrs/tests/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManagerTest.java

```diff
@@ -108,7 +108,7 @@ public void testInitialize() throws Exception {
     // Verify curator client is invoked
     Mockito.verify(mockClient).start();
     Mockito.verify(mockClient).blockUntilConnected(Mockito.anyInt(), Mockito.any(TimeUnit.class));
-    Mockito.verify(mockClient, Mockito.times(6)).createContainers(Mockito.anyString());
+    Mockito.verify(mockClient, Mockito.times(7)).createContainers(Mockito.anyString());
```

Comment:
This tests the state managers `initTree` method, which creates all `StateLocation`s and we've added a new one with `/locks`.

https://github.com/twitter/heron/blob/master/heron/statemgrs/src/java/com/twitter/heron/statemgr/FileSystemStateManager.java#L42